bigchaindb/db/utils.py

```diff
@@ -18,54 +18,91 @@ def get_conn():
-    r.db(dbname).table('backlog').index_create('transaction_timestamp', r.row['transaction']['timestamp']).run(conn)
+    r.db(dbname).table('bigchain')\
+        .index_create('block_timestamp', r.row['block']['timestamp'])\
+        .run(conn)
     # to query the bigchain for a transaction id
-    r.db(dbname).table('bigchain').index_create('transaction_id',
-                                                r.row['block']['transactions']['id'], multi=True).run(conn)
+    r.db(dbname).table('bigchain')\
+        .index_create('transaction_id',
+                      r.row['block']['transactions']['id'], multi=True)\
+        .run(conn)
+    # secondary index for payload data by UUID
+    r.db(dbname).table('bigchain')\
+        .index_create('payload_uuid',
+                      r.row['block']['transactions']['transaction']['data']['uuid'], multi=True)\
+        .run(conn)
+
+    # wait for rethinkdb to finish creating secondary indexes
+    r.db(dbname).table('bigchain').index_wait().run(conn)
+
+
+def create_backlog_secondary_index(conn, dbname):
+    logger.info(' - backlog indexes')
+    # to order transactions by timestamp
+    r.db(dbname).table('backlog')\
+        .index_create('transaction_timestamp',
+                      r.row['transaction']['timestamp'])\
+        .run(conn)
     # compound index to read transactions from the backlog per assignee
     r.db(dbname).table('backlog')\
-        .index_create('assignee__transaction_timestamp', [r.row['assignee'], r.row['transaction']['timestamp']])\
+        .index_create('assignee__transaction_timestamp',
+                      [r.row['assignee'], r.row['transaction']['timestamp']])\
         .run(conn)
 
-    # compound index to order votes by block id and node
-    r.db(dbname).table('votes').index_create('block_and_voter',
-                                             [r.row['vote']['voting_for_block'], r.row['node_pubkey']]).run(conn)
+    # wait for rethinkdb to finish creating secondary indexes
+    r.db(dbname).table('backlog').index_wait().run(conn)
 
-    # secondary index for payload data by UUID
-    r.db(dbname).table('bigchain')\
-        .index_create('payload_uuid', r.row['block']['transactions']['transaction']['data']['uuid'], multi=True)\
+
+def create_votes_secondary_index(conn, dbname):
+    logger.info(' - indexes')
+    # compound index to order votes by block id and node
+    r.db(dbname).table('votes')\
+        .index_create('block_and_voter',
+                      [r.row['vote']['voting_for_block'],
+                       r.row['node_pubkey']])\
         .run(conn)
 
     # wait for rethinkdb to finish creating secondary indexes
-    r.db(dbname).table('backlog').index_wait().run(conn)
-    r.db(dbname).table('bigchain').index_wait().run(conn)
     r.db(dbname).table('votes').index_wait().run(conn)
 
+
+def init():
+    # Try to access the keypair, throws an exception if it does not exist
+    b = bigchaindb.Bigchain()
+
+    conn = get_conn()
```

Comment:
I don't remember why we created `get_conn()`. It his the same as `b.conn`.
Since we are refactoring this maybe we can get rid off `get_conn`

@vrde thoughts?